{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c4/console.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:554:1:\n    |\n554 | import \"./console.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c4/console2.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:555:1:\n    |\n555 | import \"./console2.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c4/StdJson.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:556:1:\n    |\n556 | import \"./StdJson.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c4/Vm.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:600:1:\n    |\n600 | import \"./Vm.sol\";\n    | ^^^^^^^^^^^^^^^^^^\n\nError: Source \"@forge-std/console.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:4054:1:\n     |\n4054 | import \"@forge-std/console.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@forge-std/Script.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:4055:1:\n     |\n4055 | import {Script} from \"@forge-std/Script.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@aave-address-book/AaveGovernanceV2.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c4/c41932609f44ddab42cca7aec7e7afa41316fd4b_DeployProposal.sol:4056:1:\n     |\n4056 | import {AaveGovernanceV2, IExecutorWithTimelock} from \"@aave-address-book/AaveGovernanceV2.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
